src/etl/ReportingETL.cpp

```diff
@@ -76,7 +78,7 @@ ReportingETL::insertTransactions(
         result.nfTokensData.end(),
         [](NFTsData const& a, NFTsData const& b) {
             return a.tokenID > b.tokenID &&
-                a.transactionIndex > b.transactionIndex;
+                a.transactionIndex.value_or(0) > b.transactionIndex.value_or(0);
```

Comment:
why this change?

Reply:
because now `transactionIndex` is optional, since we cannot determine that from NFTokenPages directly. In this case however we are parsing from transactions, so the transactionIndex should always be there, otherwise something weird has happened. but i thought it would be safer to just default to 0 rather than crash

Reply:
You should check for yourself, but i think if you compare two optionals (without .value()) it would not crash. There are some rules how they are evaluated but the result should be pretty much same (confirm for yourself on godbolt.org if that's exactly what you would expect or not - don't take my word for it).